combinetf2/physicsmodels.py

```diff
@@ -0,0 +1,164 @@
+import tensorflow as tf
+
+
+class PhysicsModel:
+    """
+    A class to output histograms without any transformation.
+
+    Parameters
+    ----------
+    """
+
+    def __init__(self, indata):
+        self.name = "basemodel"  # Used for the output dictionary
+        # channel info of the resulting histogram, needed for the writing
+        self.channel_info = indata.channel_info
+        # if data histograms are stored or not, and if chi2 is calculated
+        self.has_data = True
+        self.normalize = False
```

Comment:
Why does normalize have to be a member of the base class like this?  ie does it trigger any "special" behaviour beyond just being a detail of the particular derived class that implements it?

Reply:
It's needed to calculate ndf but I've removed it here and assume it as False if not a member of the class